lib/main.js

```diff
@@ -93,6 +93,12 @@ const Hydrogen = {
     );
 
     store.subscriptions.add(
+      atom.config.onDidChange("Hydrogen.statusBarDisable", ({ newValue }) => {
```

Comment:
This should be `observe` instead of `onDidChange` so it initializes. I forgot to push that change, will do so later.